src/keycloak-theme/login/pages/error/error.component.ts

```diff
@@ -1,32 +1,34 @@
-import { Component, Input } from '@angular/core';
-import { KcClassPipe } from '../../common/pipes/classname.pipe';
-import { CommonModule } from '@angular/common';
-import { ActivatedRoute } from '@angular/router';
+import { AsyncPipe } from "@angular/common";
+import { Component, inject } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { map, Observable, Unsubscribable } from "rxjs";
+import { KC_CONTEXT } from "../../../keycloak-context.provider";
+import { KcContext } from "../../../models/KcContext";
+import { KcClassPipe } from "../../common/pipes/classname.pipe";
 
 @Component({
-  selector: 'kc-error',
+  selector: "kc-error",
   standalone: true,
-  imports: [KcClassPipe, CommonModule],
+  imports: [KcClassPipe, AsyncPipe],
   providers: [ActivatedRoute],
-  templateUrl: './error.component.html',
+  templateUrl: "./error.component.html"
 })
 export class ErrorComponent {
-  errorMessage?: string;
-  kcContext: any;
-  constructor(private route: ActivatedRoute) {}
+  kcContext: KcContext<"error.ftl"> = inject<KcContext<"error.ftl">>(KC_CONTEXT);
```

Comment:
here and on the other pages I've injected the KC_CONTEXT instead of window, I think it's cleaner

Reply:
Why is it cleaner? I don't really know about this concept.